scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -689,14 +693,22 @@ def _expm_multiply_interval(A, B, start=None, stop=None, num=None,
     A = A - mu * ident
     A_1_norm = onenormest(A) if is_linear_operator else _exact_1_norm(A)
     ell = 2
-    norm_info = LazyOperatorNormInfo(t*A, A_1_norm=t*A_1_norm, ell=ell)
+    norm_info = LazyOperatorNormInfo(t*A, A_1_norm=abs(t)*A_1_norm, ell=ell)
     if t*A_1_norm == 0:
         m_star, s = 0, 1
     else:
         m_star, s = _fragment_3_1(norm_info, n0, tol, ell=ell)
 
     # Compute the expm action up to the initial time point.
-    X[0] = _expm_multiply_simple_core(A, B, t_0, mu, m_star, s)
+    if t_0*A_1_norm == 0:
+        m_star_0, s_0 = 0, 1
+    else:
+        norm_info_0 = LazyOperatorNormInfo(
+                t_0*A,
+                A_1_norm=abs(t_0)*A_1_norm,
+                ell=ell)
+        m_star_0, s_0 = _fragment_3_1(norm_info_0, n0, tol, ell=ell)
+    X[0] = _expm_multiply_simple_core(A, B, t_0, mu, m_star_0, s_0)
 
```

Comment:
Fix the order of `m_star_0` and `m_star`, maybe even use `m_star` in both cases, avoiding additional variable names.
```suggestion

    # Compute the expm action up to the initial time point.
    if t_0*A_1_norm == 0:
        m_star_0, s_0 = 0, 1
    else:
        norm_info_0 = LazyOperatorNormInfo(
                t_0*A,
                A_1_norm=abs(t_0)*A_1_norm,
                ell=ell)
        m_star_0, s_0 = _fragment_3_1(norm_info_0, n0, tol, ell=ell)
    X[0] = _expm_multiply_simple_core(A, B, t_0, mu, m_star_0, s_0)
    
    if t*A_1_norm == 0:
        m_star, s = 0, 1
    else:
        m_star, s = _fragment_3_1(norm_info, n0, tol, ell=ell)
```

Reply:
I've fixed this ordering weirdness and added more test cases for various values of `start` and `stop`.